Python/tests/test_runner_datetime.py

```diff
@@ -81,18 +81,26 @@ def test_datetime_model(culture, model, options, context, source, expected_resul
             assert_resolution(actual_resilution_value, expected_resoulution_value)
 
 def get_assert_model_resolution(option):
-    assert_resolutions = {
-        None: assert_model_resolution_option_none,
-        'CalendarMode': assert_model_resolution_option_calendar_mode,
-        'ExtendedTypes': assert_model_resolution_option_extended_types,
-        'SplitDateAndTime': assert_model_resolution_option_split_date_and_time
-    }
-    return assert_resolutions.get(option)
```

Comment:
is it possible to use a dictionary instead of if, and use the optional parameter of the get method to return `assert_model_resolution_option_none` if the option doesn't exist?